styles/layout.css

```diff
@@ -1,4 +1,4 @@
-@media(min-width: 768px){
+/* @media(min-width: 768px){
```

Comment:
make sure you are removing corpse code (old commented code) before you submit your work